internal/authz/header_test.go

```diff
@@ -76,6 +76,23 @@ func TestParseAuthorizationHeader(t *testing.T) {
 			"path":  ".api/graphql",
 		}, logs[0].Fields)
 	})
+
+	t.Run("empty token does not raise sudo error on dotcom", func(t *testing.T) {
+		envvar.MockSourcegraphDotComMode(true)
+		defer envvar.MockSourcegraphDotComMode(false)
+
+		r := &http.Request{
+			URL:  &url.URL{Path: ".api/graphql"},
+			Body: io.NopCloser(strings.NewReader("the body")),
+		}
+		logger, _ := logtest.Captured(t)
+		_, _, err := ParseAuthorizationHeader(logger, r, `token`)
```

Comment:
since we don't actually look at the captured logs, we don't need a capturing logger here 

```suggestion
		_, _, err := ParseAuthorizationHeader(logtest.Scoped(t), r, `token`)
```